package.json

```diff
@@ -25,7 +25,7 @@
         "@leafygreen-ui/leafygreen-provider": "2.3.0",
         "@leafygreen-ui/mongo-nav": "9.2.0",
         "@mdb/consistent-nav": "1.2.15",
-        "@mdb/flora": "1.0.0",
+        "@mdb/flora": "1.1.0-rc.0",
```

Comment:
@gfunk1230 are we ok with going to PROD with an RC version of Flora? They haven't technically approved/merged my [PR](https://github.com/10gen/flora/pull/104) yet, but gave us this so we can test out integration of the Radio component

Reply:
@justinmckenzie So let's not deploy a release candidate bc there may be things in there that they being flora, wants out in the wild yet. Like anything user-facing that affects the UI and UX. While I am sure we would test the app to make sure nothing breaks, what I am afraid will happen is that some team like marketing reached out to me and asked why our nav is inconsistent with what is on mongodb.com. Hope this all makes sense, so just to be safe than sorry, I rather hold off on deploying an RC version.

Reply:
So I asked Flora's team to make sure and they said they are cool with it. So if we ever get a ping from anyone saying there are any inconsistencies we can just mention we got the ok to release the RC version and that's why there are inconsistencies